test/lib/usd/plugin/CMakeLists.txt

```diff
@@ -127,13 +127,54 @@ if(IS_MACOSX OR IS_LINUX)
     mayaUsd_install_rpath(rpath ${TARGET_NAME})
 endif()
 
+
+set(TARGET_NAME usdTestInfoImportChaser)
+add_library(${TARGET_NAME} SHARED)
+
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+target_sources(${TARGET_NAME}
+    PRIVATE
+        plugin.cpp
+        infoImportChaser.cpp
+)
+mayaUsd_compile_config(${TARGET_NAME})
+
 # -----------------------------------------------------------------------------
-# Plug Plug-in files
+# link libraries
 # -----------------------------------------------------------------------------
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mayaUsdPlugInfo.json"
```

Comment:
Here is the reason why the version check test failed. This code was removed by accident. Apply this patch to fix the test
[fix_code_removal.patch.txt](https://github.com/Autodesk/maya-usd/files/6031486/fix_code_removal.patch.txt)